backend/src/main/App.ts

```diff
@@ -23,6 +26,16 @@ createConnection()
     app.use(bodyParser.json());
     const PORT = 8000;
 
+    registerRouters(app);
+
+    function registerRouters(app: express.Express) {
+      app.get('/', (req, res) => res.send('Hello World'));
```

Comment:
We can get rid of this line now I think!

Reply:
thanks! I'll remove it!